README.md

```diff
@@ -1 +1,195 @@
-# inpycon2021-tasks
\ No newline at end of file
+# PyCon India 2021
+
+
+## Tentative Event Dates:
+
+- Conference: 16th and 17th Sep, 2021
+- Workshop: 18th Sep, 2021
+- DevSprint: 18th and 19th Sep, 2021
+
+## Conference Structure
+- Platform: Hopin
+- Communication: Zulip Chat
+- 5 tracks on stage
+    * 3 general tracks
```

Comment:
Why not just mention 4 general tracks. That would give more freedom to exercise and change the 4th later?

Reply:
Yes, make sense.